benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -121,6 +121,45 @@ void GraphicsBenchmarkApp::InitKnobs()
     pFullscreenQuadsSingleRenderpass->SetDisplayName("Single Renderpass");
     pFullscreenQuadsSingleRenderpass->SetFlagDescription("Render all fullscreen quads (see --fullscreen-quads-count) in a single renderpass.");
     pFullscreenQuadsSingleRenderpass->SetIndent(1);
+
+    GetKnobManager().InitKnob(&pRenderOffscreen, "offscreen-render", false);
```

Comment:
Nit: Consider naming this flag `enable-offscreen-render` since it might help cue users that this is a flag that a lot of other flags will depend on like `enable-skybox` and `enable-spheres`?

Reply:
Done.